modules/alloy/src/main/ts/ephox/alloy/dropdown/DropdownUtils.ts

```diff
@@ -69,12 +68,15 @@ const openF = (
 
   // TODO: Make this potentially a single menu also
   return futureData.map((tdata) => tdata.bind((data) => Optional.from(TieredMenu.sketch({
+    // Externals are configured by the "menu" part. It's called external because it isn't contained
+    // within the DOM descendants of the dropdown. You can configure things like `fakeFocus` here.
     ...externals.menu(),
 
     uid: Tagger.generate(''),
     data,
 
-    highlightImmediately: highlightOnOpen === HighlightOnOpen.HighlightFirst,
+    // Can be (i) nothing, (ii) highlight just the menu, or (iii) highlight menu and item
```

Comment:
This type of comment just screams to me that it'll get out of date in future. Does it hold any value, if not I think we should remove it?

Reply:
Nope. I can remove it.

- [ ] remove comment.